tests/metrics/test_metrics.py

```diff
@@ -728,3 +729,27 @@ def test_get_course_num_learners_completed_for_time_period(self):
             end_date=self.data_end_date,
             course_id=self.alpha_course_overview.id)
         assert actual == expected
+
+
+@pytest.mark.django_db
+class TestGetLastMonthSiteMetrics(object):
+    """
+    This test also exercises the time period getters used in
+    figures.metrics.get_last_month_site_metrics
+    """
+
+    @pytest.fixture(autouse=True)
+    def setup(self, db):
+        self.site = SiteFactory(domain='alpha.site')
+        self.expected_keys = [
```

Comment:
Can you move it to the function which uses it or if it's common you can move it to `_get_expected_keys` and use that?

Reply:
This test is for the helper method, I will add a separate Unit test for the class using it.

Reply:
setup doesn't seem the right place for `expected_keys` use separate function to get expected data.